Source/UniversalAnalyticsHttpWrapper/EventTracker.cs

```diff
@@ -54,5 +55,16 @@ public void TrackEvent(IUniversalAnalyticsEvent analyticsEvent)
 
             googleDataSender.SendData(GOOGLE_COLLECTION_URI, postData);
         }
+
+        /// <summary>
+        /// Pushes an event up to the Universal Analytics web property specified in the .config file.
+        /// </summary>
+        /// <param name="analyticsEvent">The event to be logged.</param>
+        public async Task TrackEventAsync(IUniversalAnalyticsEvent analyticsEvent)
```

Comment:
Upon thinking about this some more, I wonder if this should just be a fire-and-forget task instead of awaiting it in this method? Otherwise there probably is no actual benefit to this. What do you think?

Something like: 

```cs
        public Task TrackEventAsync(IUniversalAnalyticsEvent analyticsEvent)
        {
            var postData = postDataBuilder.BuildPostDataCollection(MEASUREMENT_PROTOCOL_VERSION, analyticsEvent);

            return googleDataSender.SendDataAsync(GOOGLE_COLLECTION_URI, postData);
        }
```